### DISCUSSION TITLE
how to remove inlay hints at the end of a line?

### DISCUSSION THREAD

**Discussion by jaredmcqueen** [LazyVim/LazyVim#1065] [Q&A] [answered]
<img width="816" alt="image" src="https://github.com/LazyVim/LazyVim/assets/3491953/46c284a6-6713-4a14-8e2a-faf2adea8709">

I can toggle inlay hints on and off just fine but I cannot figure out how to remove them from the end of the line.  Not sure what plugin controls this behavior.

**Comment by amaanq** [comment, marked as answer]
It's from rust-tools, I guess with inlay hints in nightly that option can now be removed

**Reply by jaredmcqueen** [reply]
awesome got it.  I added a `rust-tools.lua` file:

```lua
return {
  {
    "simrat39/rust-tools.nvim",
    lazy = true,
    opts = {
      tools = {
        inlay_hints = {
          auto = false,
        },
      },
    },
  },
}
```
